spec/unit/util/windows/file_spec.rb

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/env ruby
+require 'spec_helper'
+require 'puppet/util/windows/file'
```

Comment:
Yes I realized my mistake on the require fairly quickly.  I disagree that it should be in integration though.